feat: LSDV-5520: Project Creation Wizard Tip (Label Studio Community)

nick-skriabin opened the pull request:
### PR fulfills these requirements
- [ ] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [ ] Self-reviewed and ran all changes on a local instance (for bug fixes/features)



#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [ ] Frontend



### Describe the reason for change
_(link to issue, supportive screenshots etc.)_



#### What does this fix?
_(if this is a bug fix)_



#### What is the new behavior?
_(if this is a breaking or feature change)_



#### What is the current behavior?
_(if this is a breaking or feature change)_



#### What libraries were added/updated?
_(list all with version changes)_



#### Does this change affect performance?
_(if so describe the impacts positive or negative)_



#### Does this change affect security?
_(if so describe the impacts positive or negative)_



#### What alternative approaches were there?
_(briefly list any if applicable)_



#### What feature flags were used to cover this change?
_(briefly list any if applicable)_



### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)



### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit



### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

nick-skriabin committed:
Introduce Heidi tips

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5901625542

nick-skriabin committed:
Fix random tip selection

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5902201016

nick-skriabin committed:
Cookie to dismiss tips

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5903825521

nick-skriabin committed:
Caption component

nick-skriabin committed:
Fix caption styles

nick-skriabin committed:
Use caption in project creation

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5904073950

nick-skriabin committed:
Option to add URL params to the links

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5904701057

nick-skriabin committed:
Add feature flag

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5904791829

juliosgarbi committed:
fix some lint issues

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5907055489

bmartel commented on label_studio/frontend/src/components/HeidiTips/HeidiTip.tsx:
```diff
@@ -0,0 +1,69 @@
+  );
+};
+
+const HeidiLink: FC<{ link: Tip["link"] }> = ({
```
Can we move this into source order, so it is defined before HeidiTip.

juliosgarbi replied:
done! :)

juliosgarbi committed:
add new GA params

juliosgarbi committed:
add new GA params

juliosgarbi committed:
move the code to be defined before HeidiTip

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5907116794

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5907139364

bmartel reviewed (commented):
Looks good to me so far, I think this is an initial pass and there will be a follow up to fill in some of the param details correct?

juliosgarbi committed:
add link to workspace dropdown

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5907221213

nick-skriabin committed:
Fix typo

nick-skriabin committed:
Fix link behavior

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5925171669

nick-skriabin committed:
Add server_id and user_id to createURL

nick-skriabin committed:
Remove unnecessary server_id from content

nick-skriabin committed:
Remove unnecessary server and user ids from project creation tip

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5925352703

nick-skriabin committed:
Support SERVER_ID

nick-skriabin committed:
Remove placeholder

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5926601530

nick-skriabin committed:
Move server_id assignment

juliosgarbi commented:
/git merge develop
> Successfully pushed new changes:
> Merge remote-tracking branch 'origin/develop' into fb-lsdv-5520/project-creation-tip (80f6a1c33ff16a02eae17516f5b915dec8cefbb9)
>
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/5939660547)

juliosgarbi committed:
Merge remote-tracking branch 'origin/develop' into fb-lsdv-5520/project-creation-tip
